### PR TITLE
Add SwiftUI Support

### PR DESCRIPTION
**Pull request by gongzhang**
## Goal
- Add SwiftUI Support (iOS)
- Add SwiftUI Example (iOS) 

## Checklist

- [x] Testing in `iOS`
- [x] Installed correct via Swift Package Manager
- [ ] Installed correct via Cocoapods

### CODE REVIEW

File: Sources/SPConfetti/SPConfetti.swift [hunk 1]
```diff
@@ -39,13 +39,16 @@ public class SPConfetti {
...
     @available(iOSApplicationExtension, unavailable)
-    public static func startAnimating(_ animation: SPConfettiAnimation, particles: [SPConfettiParticle]) {
+    public static func startAnimating(_ animation: SPConfettiAnimation, particles: [SPConfettiParticle], in window: UIWindow? = nil) {
         shared.view.animation = animation
         shared.view.particles = particles
-        guard let window = UIApplication.shared.keyWindow else { return }
-        if shared.view.superview == nil {
+        guard let window = window ?? UIApplication.shared.keyWindow else { return }
+        if let superview = shared.view.superview, superview == window {
+            superview.bringSubviewToFront(shared.view)
```
Comment (gongzhang) [inline Sources/SPConfetti/SPConfetti.swift:50]: For https://github.com/ivanvorobei/SPConfetti/issues/2

**Comment by ivanvorobei** [comment]
Thanks for your work! Can you add section in readme with description how use it?

**Comment by gongzhang** [comment]
@ivanvorobei README is updated. 😄

**Comment by ivanvorobei** [comment]
Thanks for your work!

**Pull request merged**